src/app/app.module.ts

```diff
@@ -3,11 +3,21 @@ import { NgModule }      from '@angular/core';
 
 import { AppComponent }          from './app.component';
 import { CaesarPortalComponent } from './caesar-portal/caesar-portal.component';
+import { GroupAreaComponent } from './caesar-portal/group-area/group-area.component';
+import { GroupInfoComponent } from './caesar-portal/group-area/group-info/group-info.component';
+import { GroupStudentsComponent } from './caesar-portal/group-area/group-students/group-students.component';
+import { GroupScheduleComponent } from './caesar-portal/group-area/group-schedule/group-schedule.component';
+import { GroupNotificationsComponent } from './caesar-portal/group-area/group-notifications/group-notifications.component';
 
 @NgModule({
   declarations: [
     AppComponent,
-    CaesarPortalComponent
+    CaesarPortalComponent,
+    GroupAreaComponent,
```

Comment:
This is not a correct place for all this declarations, you should define components in the modules that will use it, not the app one